cmd/integration/integration.go

```diff
@@ -955,6 +959,7 @@ type testFunc func(*client.Client)
 func addFlags(fs *pflag.FlagSet) {
 	fs.IntVar(
 		&maxConcurrency, "max-concurrency", -1, "Maximum number of tests to be run simultaneously. Unlimited if set to negative.")
+	fs.StringVar(&etcdServer, "etcd-server", "", "etcd server URL.")
```

Comment:
this also needs port, right? (maybe ... URL in http://host:port format)?

Also, how about being consistent with other places and use "etcd-servers" instead to handle multiple etcd servers too:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/server.go#L214